packages/transformer/src/IModelCloneContext.ts

```diff
@@ -28,7 +29,7 @@ const loggerCategory: string = TransformerLoggerCategory.IModelCloneContext;
 export class IModelCloneContext extends IModelElementCloneContext {
 
   private _refTypesCache = new ECReferenceTypesCache();
-  private _aspectRemapTable = new Map<Id64String, Id64String>();
+  private _aspectRemapTable = new BigMap<Id64String>();
```

Comment:
Regarding this, while I would defer the work to another PR, but I did realize we probably can in javascript compress the remap table with runs pretty easily, by just inlining the two numbers directly into an array and running an adjusted binary search on that array.

Reply:
I think a better approach would be to minimize how many aspects are being saved in this table. Aspects cannot be referenced by other elements in EC schemas and as far as I know elements should reference owning element instead as aspect is sort of extension of element. So this table should only be needed when partially inserted aspect is fully completed so that transformer could update it. After that it should be safe to remove aspect's record from the table and in case aspect doesn't have pending references it should be able to entirely skip storing it in this remap table.

But I agree that this should be a separate PR and removed this change from here.

Reply:
yes you're right that should be possible, resolving